### ISSUE TITLE
Providing more clear visual indicator for keyboard navigation

### ISSUE THREAD

**Issue by bick-jp** [internetarchive/openlibrary#7857]
For users who rely on a keyboard for navigation (typically using the tab key), it's nice to have a more clear visual indicator of the element that currently has keyboard focus.

### Describe the problem that you'd like solved

This is an ok example. I think it's clear that the focus is on "open library" because of the blue-colored outline.
<img width="386" alt="Screen Shot 2023-05-11 at 1 17 44 PM" src="https://github.com/internetarchive/openlibrary/assets/8709562/9d138f0b-0973-43fa-898c-70b9fd3c8784">

Here are some examples that keyboard users might get confused for navigation.
- The focus is on "All," but the outline color is grey and hard to see.
<img width="394" alt="Screen Shot 2023-05-11 at 1 14 29 PM" src="https://github.com/internetarchive/openlibrary/assets/8709562/00b40047-b92f-44ef-85b0-d56fe6482005">

- The focus is on the search bar, but there is no outline, and it's hard to see.
<img width="383" alt="Screen Shot 2023-05-11 at 1 14 34 PM" src="https://github.com/internetarchive/openlibrary/assets/8709562/278e5b06-f688-485d-b2e4-c4a250b33a09">

- The focus is on the right arrow button, but the button is light blue so that the darker-blue focus is hard to see.
<img width="384" alt="Screen Shot 2023-05-11 at 1 14 53 PM" src="https://github.com/internetarchive/openlibrary/assets/8709562/6e622cea-79f0-4c0c-88dc-7dc9d14d1a13">

- The focus is on the left arrow button, but there is no indicator that the focus is on this element. It is because, in this case, the button is disabled since there is no item further on the left.
<img width="389" alt="Screen Shot 2023-05-11 at 1 15 05 PM" src="https://github.com/internetarchive/openlibrary/assets/8709562/cf1acd79-be4c-4351-944a-26a7699f2970">

### Proposal & Constraints

Adding a more clear visual indicator for keyboard focus. It could be a consistent outline appearance (e.g., missing colored outline in the search bar), avoiding the use of the same color for the element and outline (e.g., blue color button on blue color outline), or even adding background color for the focused element as example below.

<img width="425" alt="Screen Shot 2023-05-11 at 1 26 27 PM" src="https://github.com/internetarchive/openlibrary/assets/8709562/b43702d8-774b-4354-a4fe-d3350b443dc5">

[WebAIM webpage](https://webaim.org/techniques/keyboard/) is a good example.

**Comment by davidscotson** [comment]
For the slider arrows, the library "slick" is used:

https://kenwheeler.github.io/slick/

This is a somewhat unmaintained legacy library, but some users of it have forked it to improve the accessability:

https://accessible360.github.io/accessible-slick/

But, it's a drop in replacement for version 1.8 and OpenLibrary currently uses 1.6, which aren't as directly swappable. If work was done to switch it might be worth looking to use a completely different alternative, as carousels are tricky to make accessible and mobile friendly. But that's obviously a bigger task.
